adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -87,6 +87,7 @@
     private MqttServer server;
     private MqttServer insecureServer;
     private AuthHandler<MqttContext> authHandler;
+    private final long connectionLimit = Long.MAX_VALUE; // TODO make configurable
```

Comment:
can you add the configuration property right now?

Reply:
Sure. What do you prefer: adding it to `ProtocolAdapterProperties` or creating a new `MqttProtocolAdapterProperties` (and analogously the other protocol adapters)? I tend to the former one, even if it might not be used by e.g. the HTTP adapter.

Reply:
I also think that `ProtocolAdapterProperties` would be the right place.